src/aequitas_cli/aequitas_audit.py

```diff
@@ -6,6 +6,7 @@
 from aequitas.bias import Bias
 from aequitas.fairness import Fairness
 from aequitas.group import Group
+from aequitas.plotting import Plotting
```

Comment:
I believe you've renamed `Plotting` to `Plot` :smile:

Reply:
Yeah, caught that as well. I'm working on one more test for generated markdown and have made the change in that branch, but it isn't ready for a PR yet--is it kosher to make the update on the master branch (since plotting branch has now been closed?)